src/AssemblySharedInfoGenerator/AssemblySharedInfo.cs

```diff
@@ -1,67 +0,0 @@
-﻿using System;
```

Comment:
Please revert the change, you are deleting this file as part of this PR

Reply:
In the future, if you accidentally included change to this file, you can commit what the number was before, then it will appear you have not touched this file in PR